docs/terminal/brisbane.md

```diff
@@ -92,7 +92,8 @@ These standard runway assignments may be modified strategically by the BAS or BF
 ### Independent Visual Approach Phraseology at Night
 *"CLEARED INDEPENDENT VISUAL APPROACH RUNWAY (number), NOT BELOW (altitude) UNTIL ESTABLISHED ON THE PAPI (or GLIDEPATH)"*
```

Comment:
While we're here, would you mind quickly changing this to a paragraph saying that it's not below the MSA until established? Maybe we could just change the existing text here to an example